web/modules/custom/joinup_front_page/joinup_front_page.permissions.yml

```diff
@@ -0,0 +1,13 @@
+'feature entity':
+  title: 'Feature entity'
+  description: 'Promote an entity as featured in the website.'
+'unfeature entity':
+  title: 'Unfeature entity'
+  description: 'Removes an entity node from the featured ones in the website.'
+
+'pin entity':
+  title: 'Pin entity'
+  description: 'Promote an entity as pinned in the website.'
```

Comment:
```
Pins an entity to the front page.
```

Reply:
You know, even though this is a purely technical description with no impact on the user, I like it really too much for its clear message :) I am reading the current one and can only think about the frustration it might cause!